sdk/network/Azure.ResourceManager.Network/src/Generated/BgpConnection.cs

```diff
@@ -23,7 +23,8 @@ namespace Azure.ResourceManager.Network
     public partial class BgpConnection : ArmResource
     {
         private readonly ClientDiagnostics _clientDiagnostics;
-        private readonly VirtualHubBgpConnectionsRestOperations _restClient;
+        private readonly VirtualHubBgpConnectionRestOperations _virtualHubBgpConnectionRestClient;
```

Comment:
So here there are two rest clients... I guess it's due to missing an `s` in the operation group.

- Here we use [VirtualHubBgpConnection_Get](https://github.com/Azure/azure-rest-api-specs/blob/e729721df20047dcb4fdb675d234b6821f1a4f81/specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/virtualWan.json#L4564)
- But here we use [VirtualHubBgpConnections_ListAdvertisedRoutes](https://github.com/Azure/azure-rest-api-specs/blob/e729721df20047dcb4fdb675d234b6821f1a4f81/specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/virtualWan.json#L4859)

How do we deduce the rest client we want? Through operation groups?